website_no_crawler/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2016 B-informed B.V.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Alter robots.txt disallow indexing",
+    "summary": "Disables robots.txt for indexing by webcrawlers like Google",
+    "license": "AGPL-3",
+    "version": "13.0.1.0.0",
+    "website": "https://github.com/OCA/website",
+    "author": "B-Informed B.V., Odoo Community Association (OCA)",
```

Comment:
```suggestion
    "author": "B-Informed B.V., Tecnativa, Odoo Community Association (OCA)",
```

Reply:
Not really if the contribution is very little.